src/app/applications/application-change-organization-dialog/application-change-organization-dialog.component.ts

```diff
@@ -0,0 +1,110 @@
+import { OrganisationService } from "@app/admin/organisation/organisation.service";
+import { PermissionResponse } from "@app/admin/permission/permission.model";
+import { PermissionService } from "@app/admin/permission/permission.service";
+import { Application, UpdateApplicationOrganization } from "@applications/application.model";
+import { ApplicationService } from "@applications/application.service";
+import { TranslateService } from "@ngx-translate/core";
+import { ApplicationDialogModel } from "@shared/models/dialog.model";
+import { SharedVariableService } from "@shared/shared-variable/shared-variable.service";
+import { ReplaySubject, Subscription } from "rxjs";
+
+@Component({
+  selector: "app-change-organization-dialog",
+  templateUrl: "./application-change-organization-dialog.component.html",
+  styleUrls: ["./application-change-organization-dialog.component.scss"],
+})
+export class ApplicationChangeOrganizationDialogComponent implements OnInit {
+  public applicationsSubscription: Subscription;
+  public permissionsSubscription: Subscription;
+  public organizationsSubscription: Subscription;
+  public application: UpdateApplicationOrganization;
+  public permissions: PermissionResponse[];
+  public organizations: Organisation[];
+  public filteredPermissionsMulti: ReplaySubject<PermissionResponse[]> = new ReplaySubject<PermissionResponse[]>(1);
+  public filteredOrganizations: ReplaySubject<Organisation[]> = new ReplaySubject<Organisation[]>(1);
+
+  constructor(
+    private applicationService: ApplicationService,
+    public translate: TranslateService,
+    private permissionService: PermissionService,
+    private organizationService: OrganisationService,
+    private sharedVariableService: SharedVariableService,
+    private snackBar: MatSnackBar,
+    private dialog: MatDialogRef<ApplicationChangeOrganizationDialogComponent>,
+    @Inject(MAT_DIALOG_DATA) public dialogModel: ApplicationDialogModel
+  ) {
+    this.application = {
+      organizationId: this.dialogModel.organizationId ?? this.sharedVariableService.getSelectedOrganisationId(),
+      permissionIds: [],
+    };
+  }
+
+  ngOnInit(): void {
+    this.translate.use("da");
+    if (this.dialogModel.id) {
+      this.getApplication(this.dialogModel.id);
+    }
+    this.getOrganizations();
+    this.getPermissions();
+  }
+
+  getApplication(id: number): void {
+    this.applicationsSubscription = this.applicationService.getApplication(id).subscribe((application: Application) => {
+      this.application.permissionIds = application.permissionIds;
+    });
+  }
+
+  getOrganizations() {
+    this.organizationsSubscription = this.organizationService.getMinimal().subscribe(res => {
+      this.organizations = res.data;
+      this.filteredOrganizations.next(this.organizations.slice());
+    });
+  }
+
+  getPermissions() {
+    this.permissionsSubscription = this.permissionService.getPermissions(1000, 0).subscribe(res => {
```

Comment:
Dit get permissions her henter kun for de organisationer hvor brugeren har userAdmin rettighed, hvilket vil sige at man ikke altid kan se alle da permissions ikke er herakiske. (Den bør nok laves om til at lave et fetch for den givne organisations permissions hvis man er applikations admin). Alternativt skal vi lige diskutere den rettighed her med Mogens